scio-extra/src/main/scala/com/spotify/scio/extra/csv/CsvIO.scala

```diff
@@ -17,27 +17,26 @@
 
 package com.spotify.scio.extra.csv
 
-import java.io.{Reader, Writer}
-import java.nio.channels.{Channels, WritableByteChannel}
-import java.nio.charset.StandardCharsets
+import com.spotify.scio.ScioContext
 import com.spotify.scio.coders.Coder
 import com.spotify.scio.io._
-import com.spotify.scio.ScioContext
-import com.spotify.scio.util.ScioUtil
-import com.spotify.scio.util.FilenamePolicySupplier
+import com.spotify.scio.util.{FilenamePolicySupplier, ScioUtil}
 import com.spotify.scio.values.SCollection
-import kantan.csv._
-import kantan.codecs.compat._
```

Comment:
IntelliJ probably tells this import is unused for 2.13, but it is required for +compile with 2.12

Reply:
Sounds good. Re-adding it